package.json

```diff
@@ -32,6 +32,7 @@
     "lodash": "4.16.2",
     "minimist": "1.1.1",
     "pm2": "2.2.3",
-    "replace-in-file": "0.2.1"
+    "replace-in-file": "0.2.1",
+    "redis-server": "1.1.0"
```

Comment:
On travis that seems to start the redis server which we don't want. I'll add a test to ping the server